src/_includes/landing/hero.njk

```diff
@@ -1,27 +1,46 @@
+    <div class="w-full">
+        <div class="flex justify-between w-full items-center">
+            <h1 class="text-white text-4xl sm:text-5xl font-extrabold leading-tight">
+                PYCON <br/>
+                <span class="text-white">INDIA 2025</span>
+            </h1>
+        </div>
+        <p class="text-white text-md sm:text-lg max-w-xl leading-relaxed">
+            PyCon India 2025 is the premier conference for Python enthusiasts and professionals, offering an unparalleled
+            opportunity to dive deep into the world of Python and explore its limitless potential. Stay tuned for updates!
+        </p>
+        <div class="flex flex-wrap gap-4">
+            {{ button(text="Get Tickets", url=env.baseUrl + "tickets", lime_bg=true)}}
+            {{ button(text="View Schedule", url=env.baseUrl + "program/schedule", lime_bg=true)}}
         </div>
     </div>
-    <p class="text-white text-md sm:text-lg max-w-xl leading-relaxed">
-        PyCon India 2025 is the premier conference for Python enthusiasts and professionals, offering an unparalleled
-        opportunity to dive deep into the world of Python and explore its limitless potential. Stay tuned for updates!
-    </p>
-    <div class="flex flex-wrap gap-4">
-        {{ button(text="Get Tickets", url=env.baseUrl + "tickets", lime_bg=true)}}
-        {{ button(text="View Schedule", url=env.baseUrl + "program/schedule", lime_bg=true)}}
+    <div class="grid w-full grid-cols-1 gap-x-1 gap-y-2 lg:grid-cols-12 lg:auto-rows-auto">
+        <div class="pb-2 lg:row-start-1 lg:col-start-2 lg:col-end-12 xl:col-start-3 xl:col-end-11">
+            {% call billboard(theme="", title="12 Sep - 15 Sep") -%}
+            {%- endcall %}
+        </div>
+        <div class="lg:row-start-2 lg:col-start-1 lg:col-end-9 xl:col-start-1 xl:col-end-7">
+            {% call billboard(theme="purple", title="12 Sep - Workshop", body="Christ University, Bangalore") -%}
+            {%- endcall %}   
+        </div>
+        <div class="relative overflow-visible lg:row-start-2 lg:col-start-9 lg:col-end-13">
+            <img 
+                src="{{ env.baseUrl }}img/assets/timeline.svg" 
+                alt="Calendar"
+                class="absolute bottom-0 right-0 hidden w-full translate-x-8 translate-y-2 max-h-[140px] object-contain pointer-events-none lg:block z-0"
+            />            
+        </div>
+        <div class="lg:row-start-3 lg:col-start-3 lg:col-end-11 xl:col-start-4 xl:col-end-10">
+            {% call billboard(theme="red", title="13-14 Sep - Conference", body="NIMHANS, Bangalore") -%}
+            {%- endcall %}
+        </div>
+        <div class="lg:row-start-4 lg:col-start-5 lg:col-end-13 xl:col-start-7 xl:col-end-13">
+            {% call billboard(theme="blue", title="15 Sep - DevSprint", body="Google Kyoto Building, Bangalore") -%}
+            {%- endcall %}
+        </div>
```

Comment:
Bangalore => Bengaluru
Christ University => St. Francis College
NIMHANS => NIMHANS Convention Centre
Google Kyoto Building, Bangalore => To be announced

Reply:
Created a new `src/_data/event_schedule.json` file to source content. Updated values as suggested.